Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.cxx

```diff
@@ -2061,11 +2089,22 @@ ::CreateTransformedClipper(vtkMRMLTransformNode *tnode, vtkMRMLModelNode::MeshTy
     {
     if (type == vtkMRMLModelNode::UnstructuredGridMeshType)
```

Comment:
Now that we have clipping method member variable, use that variable to decide between vtkClipDataSet/vtkExtractGeometry. Probably you don't even have to check what's the mesh type.